ec_api/apps/users/forms.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+from django.contrib.auth import get_user_model
+
+
+class RegisterForm(forms.ModelForm):
+    """
+    Registration form for a User
+    """
+
+    class Meta:
+        model = get_user_model()
+        fields = ["email"]
+        error_messages = {
+            "email": {
+                "unique": "User with this email has already registered. Do you need to log in?"
+            }
+        }
+
+    def save(self, commit=True):
+        """
+        Password field is not used to set unusable password before saving
```

Comment:
Possible small typo: should 'Password field is not used to set' be 'Password field is not used, so set'? 
If not, I don't understand the comment (but that will be due to me not knowing about handling passwords in django... :grinning: )

Reply:
Yep... that sentence makes no sense at all 😆! I'll update to something that does, thanks